### PR TITLE
Update Vagrant sandbox

### PR DESCRIPTION
**Pull request by pazaan**
- Update to the currently used package versions
- Include `platform` in the services to run

@darinkrauss, are you happy with the way the `platform` additions look?

### CODE REVIEW

File: runservers [hunk 1]
```diff
@@ -292,6 +292,20 @@ tp_blip() {
     echo "Blip started at http://localhost:$PORT"
 }
 
+# platform
+tp_platform() {
+    echo "Starting platform"
+    export GOPATH=/tidepool/platform
+    cd $GOPATH/src/github.com/tidepool-org/platform
+    . ./.env
+    _bin/dataservices/dataservices >> /tidepool/platform_server.log 2>&1 &
```
Comment (darinkrauss) [inline runservers]: We'll also need to start `userservices` at some point. It starts to be necessary with the delete user (and all associated data) functionality. So really, we need a `tp_platform_dataservices` and `tp_platform_userservices`, I think.

File: get_current_tidepool_repos.sh [hunk 2]
```diff
@@ -41,7 +41,20 @@ get_one_tidepool_repo()
     fi
 }
 
+setup_platform()
+{
+    echo "*** platform ***"
+    mkdir -p platform
+    export GOPATH=${PWD}/platform
+    # Ignore the "no buildable Go source files" warning
+    go get github.com/tidepool-org/platform 2>&1 | grep -v "no buildable Go source files"
```
Comment (darinkrauss) [inline get_current_tidepool_repos.sh]: Shouldn't this use `git clone` rather than `go get`?

Comment (pazaan) [inline get_current_tidepool_repos.sh]: I'm not really in a position to answer that, so I'll let you make the call.
I was just following the initial process outlined in the platform README.md.

Comment (darinkrauss) [inline get_current_tidepool_repos.sh]: Well, I suppose it really doesn't matter. It should work either way.

**Comment by pazaan** [comment]
@darinkrauss I've made some changes to address this PR. Can you please review them?

File: Vagrantfile [hunk 3]
```diff
@@ -14,25 +14,36 @@ Vagrant.configure(2) do |config|
   config.vm.network "forwarded_port", guest: 8009, host: 8009
   config.vm.network "forwarded_port", guest: 9122, host: 9122
   config.vm.network "forwarded_port", guest: 3004, host: 3004
+  config.vm.network "forwarded_port", guest: 8077, host: 8077 # dataservices
+  config.vm.network "forwarded_port", guest: 8077, host: 8078 # userservices
```
Comment (darinkrauss) [inline Vagrantfile]: Should guest port for userservices be 8078?

Comment (pazaan) [inline Vagrantfile]: Yeah, it should. Fixed :)

**Pull request merged**